bin/lang-js/src/function_kinds/resolver_function.ts

```diff
@@ -127,7 +138,7 @@ const isQualification = (value: unknown): TypeCheckResult => {
     };
   }
 
-  if (!qualificationStatuses.includes(value.result)) {
+  if (!qualificationStatuses.includes(value.result as string)) {
```

Comment:
I do wonder if we can use an else statement, so TS "knows"/proves that `value.result` is a string, without casting it.. but NBD.. just trying to get the most of out TS checks